s/children.go

```diff
@@ -69,23 +76,34 @@ func startChildren(
 
 	// Start children
 	for _, cs := range spec.order.SortStart(spec.children) {
-		startTime := time.Now()
-		c, err := cs.Start(runtimeName, notifyCh)
+		startedTime := time.Now()
+		ch, startErr := cs.Start(runtimeName, notifyCh)
+
 		// NOTE: The error handling code bellow gets executed when the children
 		// fails at start time
-		if err != nil {
-			cRuntimeName := strings.Join([]string{runtimeName, cs.Name()}, "/")
-			eventNotifier.ProcessStopped(cRuntimeName, startTime, err)
+		if startErr != nil {
+			cRuntimeName := strings.Join([]string{runtimeName, cs.Name()}, childSepToken)
+			// NOTE: we only notify when child is a worker because sub-trees
+			// supervisors are responsible of their own notification
+			eventNotifier.ProcessStartFailed(cRuntimeName, startErr)
```

Comment:
The comment above is out of date? The worker check is commented out.

Reply:
Yes, that is outdated, thanks for catching this, removing the comment